src/zarr/codecs/zstd.py

```diff
@@ -44,12 +42,12 @@ class ZstdCodec(BytesBytesCodec):
 
     def __init__(self, *, level: int = 0, checksum: bool = False) -> None:
         # numcodecs 0.13.0 introduces the checksum attribute for the zstd codec
-        _numcodecs_version = Version(numcodecs.__version__)
-        if _numcodecs_version < Version("0.13.0"):
-            raise RuntimeError(
-                "numcodecs version >= 0.13.0 is required to use the zstd codec. "
-                f"Version {_numcodecs_version} is currently installed."
-            )
+        # _numcodecs_version = Version(numcodecs.__version__)
+        # if _numcodecs_version < Version("0.13.0"):
+        #     raise RuntimeError(
+        #         "numcodecs version >= 0.13.0 is required to use the zstd codec. "
+        #         f"Version {_numcodecs_version} is currently installed."
+        #     )
```

Comment:
Temporary change; these lines will be uncommented when we either have a solution for the `numcodecs` version being fetched in the CI job or we create yet another alpha with an updated version of `numcodecs`.